src/main/java/bot/exception/IncompleteBotException.java

```diff
@@ -0,0 +1,13 @@
+package bot.exception;
+
+public class IncompleteBotException extends BotException {
```

Comment:
Good use of different exceptions to categorise the various types of exceptions you might encounter.